css/animations.css

```diff
@@ -1073,3 +1073,97 @@
     transform: rotate(45deg) scale(0.5);
   }
 }
+
+/**
+* Animation #17 ----------------------------------------------------
+*/
+.animation-17 {
+  background-color: #333333;
+  display: flex;
+  align-items: center;
+  justify-content: center;
+}
+
+.animation-17 .wrapper {
+  width: 150px;
+  height: 150px;
+  position: relative;
+}
+
+.animation-17 .wrapper > span {
+  position: absolute;
+  width: 50px;
+  height: 50px;
+  background-color: #fd6c64;
+  border-radius: 4px;
+  animation: animation17 3s linear infinite;
+}
+
+.animation-17 .wrapper > span:nth-child(2) {
+  animation-delay: 1s;
+  background-color: #55ddb9;
+}
+
+.animation-17 .wrapper > span:nth-child(3) {
+  animation-delay: 2s;
+  background-color: #1f78b0;
+}
+
+@keyframes animation17 {
```

Comment:
We need to be sure all the names of the animations correspond to the name of the class. I can see that in some other cases it does not happen, we need to fix it.